src/tabs/Batch_Gift_Entry.tab

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomTab xmlns="http://soap.sforce.com/2006/04/metadata">
+    <label>Batch Gift Entry</label>
+    <mobileReady>false</mobileReady>
+    <motif>Custom41: Stack of Cash</motif>
```

Comment:
The managed package docs say "The tab style for a custom tab must be unique within your app." so I suspect we need to use a different icon here, since Stack of Cash is already used for Payments.